.github/workflows/i18n_server_list_update.yaml

```diff
@@ -0,0 +1,37 @@
+name: i18n Updater
+# This Workflow will periodically go in:
+# 1. Update extras.xliff with any new servers names Mullvad may have added
+# 2. File a pr with the changes to extras.xliff
+
+on:
+  schedule:
+    - cron: "0 7 * * *" # Every Day 7:00 UTC
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  main:
+    name: Update server name strings
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
This is just in line with the versions used by our other Github actions. I will send a commit where I update this and the create-pull-request actions everywhere just to keep the pattern.